### DISCUSSION TITLE
Bit confused in github desktop

### DISCUSSION THREAD

**Discussion by SulthanNK** [community/community#23794] [New to GitHub]
<p>Using Github in the browser everything is fine. But, while using the Github Desktop I facing lots of issue and confusion.</p>
<p>While, inside the repository, I can’t back to the home page or the main page that shows the list of my all repositories.</p>
<p><strong>Disappearing of the repositories.</strong> Yes, while I’m using a current repository I can’t able to view my other repositories even I search. It shows only one or two.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>In GitHub Desktop, there isn’t really a home page to go back to.</p>
<p>While you’re in a repository its name will be in the top left corner of Desktop, in the section called “Current Repository”.</p>
<p>If you click the arrow in that section it will show you a list of the repositories that you have in GitHub Desktop. This is perhaps where you are getting confused, as you might expect it to show you <strong>all</strong> of your repos from GitHub? Actually, it’s only showing you the ones you have locally.</p>
<p>If you click the “Add” button that is next to the filter box, and then “Clone Repository”, that <strong>will</strong> show you a list of all the repositories you have access to on GitHub - not just your own, but also repos you are a collaborator on, or have access to via an organization!</p>
<p>You can filter the list to quickly find the specific repo you want, and then clone it down to GitHub Desktop, and then it will be visible in the list you are used to looking at.</p>
<p>I hope that clarifies things!</p>
